project-lisa/source-code/client/src/github.ts

```diff
@@ -0,0 +1,27 @@
+import { Octokit } from "octokit"
+
+/**
+ * The path used for proxy purposes.
+ *
+ * The server will proxy all requests to this path to the github API.
+ */
+export const PATH = "/github-proxy/"
```

Comment:
Not needed anymore. The github proxy is also just lisa/server.

Reply:
Reverting my remark: Using `load('", { corsProxy: "link.to/proxy"})` is the API that will be used. No manual export is required anymore.